tools/splat.yaml

```diff
@@ -401,71 +407,84 @@ segments:
     files:
     - [0x316f30, c, code_316f30]
   - type: code
+    ld_name: world_partner_goombario
     start: 0x317020
     vram: 0x802BD100
     files:
-      - [0x317020, c]
-      - [0x317b60, bin]
+      - [0x317020, c, world/partner/goombario]
+      - [0x317b60, .data, world/partner/goombario]
+      - [0x317DA0, bin, world/partner/goombario] # rodata containing a double and a jumptable
```

Comment:
since we know this is rodata, use the ``rodata`` type in n64splat; this will auto-disassemble the rodata based on the asm in the code associated with the file

Reply:
I don't think we should require the use of this necessarily..the data disassembly is still in the very early stages, so I don't think it makes sense to use it unless you're going to leverage it (migrate the rodata, move it into c, etc)

Reply:
iirc I tried it but it didn't work